diff --git a/pnc_cli/swagger_client/apis/bpm_api.py b/pnc_cli/swagger_client/apis/bpm_api.py
index 0586a1d..bdea2d1 100644
--- a/pnc_cli/swagger_client/apis/bpm_api.py
+++ b/pnc_cli/swagger_client/apis/bpm_api.py
@@ -352,7 +352,7 @@ class BpmApi(object):
                                             body=body_params,
                                             post_params=form_params,
                                             files=files,
-                                            response_type=None,
+                                            response_type=int,
                                             auth_settings=auth_settings,
                                             callback=params.get('callback'))
         return response
